src/auth/operations/refresh.ts

```diff
@@ -78,30 +96,32 @@ async function refresh(incomingArgs: Arguments): Promise<Result> {
     args.res.cookie(`${config.cookiePrefix}-token`, refreshedToken, cookieOptions);
   }
 
+  let response: Result = {
+    user,
+    refreshedToken,
+    exp,
+  };
+
   // /////////////////////////////////////
   // After Refresh - Collection
   // /////////////////////////////////////
 
   await collectionConfig.hooks.afterRefresh.reduce(async (priorHook, hook) => {
     await priorHook;
 
-    args = (await hook({
+    response = (await hook({
       req: args.req,
       res: args.res,
       exp,
       token: refreshedToken,
-    })) || args;
+    })) || response;
```

Comment:
Another question on this one: if there are multiple afterRefresh hooks, will the first one receive `token` as property and the other ones `refreshToken`? (I'm not sure how hooks behave exactly. Do they all receive the same input or do they pipe from one to the next?